qemu/tests/cfg/nested_system_reset.cfg

```diff
@@ -0,0 +1,12 @@
+- nested_system_reset:
+    only Linux
+    type = nested_system_reset
+    virt_test_type = qemu
+    no RHEL.7, RHEL.8.0, RHEL.8.1, RHEL.8.2
+    reboot_cmd = "system_reset"
+    qemu_cmd = "/usr/libexec/qemu-kvm &"
+    pgrep_cmd = "pgrep qemu-kvm"
+    HostCpuVendor.amd:
+        cpu_model_flags += ",+svm"
+    HostCpuVendor.intel:
+        cpu_model_flags += ",+vmx"
```

Comment:
Also check `kvm_probe_module_parameters = "nested=1"`?

Reply:
I think this is optional. For x86, Intel and AMD are both nested=1 by default. No sure about other archs ?